### ISSUE TITLE
Documentation on working with EF Core across environments & in a separate project from the website

### ISSUE THREAD

**Issue by jemayn** [umbraco/UmbracoDocs#5595]
## What type of issue is it? (Choose one - delete the others)
Missing documentation

## Describe the issue
I've recently hit a couple of issues when using the new Umbraco EF Core setup for managing custom data.

I wanted to make a PR to note some of the challenges and solutions I've found, but after starting I realized that there isn't any actual pages for it. Currently all mentions of EF Core are in a [tutorial](https://docs.umbraco.com/umbraco-cms/tutorials/getting-started-with-entity-framework-core) - which is fine for what it is, but I don't think it makes sense in a tutorial to start talking about setting connection strings for cross environment deployments and how to manage your EF Core entities when they are in a seperate project to the website project, for things like that it should probably have a reference section somewhere?

In short - I am happy to contribute some documentation but unsure where 🙂

**Comment by jonat123** [comment]
@jemayn thanks for raising this issue! 

I think it makes total sense to add documentation other than a tutorial for EF core in Umbraco :-) 

I think it could be in the extending section maybe? 🤔 
As the EF Core is used for creating custom databases for your projects, so you are in a way extending your project when using it 🙂

Let me know what you think 😄

**Comment by sofietoft** [comment]
Hi @jemayn 👋 

Can you help me verify whether this issue is still relevant?

**Comment by jemayn** [comment]
It is, the "right way" of doing it in Core post v12 I think is really convuluted. Been waiting on https://github.com/umbraco/Umbraco-CMS/pull/15186 before wrapping up my PR but it seems to be somewhat stuck.
